### PR TITLE
feat/add-axlusdc-alt-pool-balance

### PR DESCRIPTION
**Pull request by hannosuke16**
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
5. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/platypus-finance exports TVL: 
        
 
 
    avax-staking              1.75 M
    avax                      35.47 M
    staking                   1.75 M
    
    total                    35.47 M

File: projects/platypus-finance/constants.js [hunk 1]
```diff
@@ -258,4 +263,13 @@ module.exports = {
       },
     ],
   },
+  AXLUSDC: {
```
Comment (devisfun201) [inline projects/platypus-finance/constants.js:267]: need to add `ALT_USDC_AXLUSDC_LP` under `USDC` addresses as well.

**Review by devisfun201 — CHANGES_REQUESTED** [review]
need to add ALT_USDC_AXLUSDC_LP under USDC as well.

**Comment by llamatester** [comment]
The adapter at projects/platypus-finance exports TVL: 
        
 
 
    avax-staking              1.86 M
    avax                      36.46 M
    staking                   1.86 M
    
    total                    36.46 M

**Review by ptpdefi — APPROVED** [review]

**Review by devisfun201 — APPROVED** [review]

**Pull request merged**